### PR TITLE
Add game section, Fix anchor bugs 

### LINKED ISSUE #19: Re-enable game page

### PR DESCRIPTION
**Pull request by R-Jim**
Game section should look like this
For desktop view
Animation
For mobile view 
Animation2

### CODE REVIEW

**Review by DuongTyler — CHANGES_REQUESTED** [review]
we might want to make the game card take up the entirety of the horizontal space since there isn't going to be more than one for this site. Also, it might be better to have the title and description both be on the bottom of the card, having title + buttons on top and description on bottom seems jarring to me...

**Comment by DuongTyler** [comment]
or perhaps we can just make a dedicated section of the page and hard-code the stuff in there instead of using the gamesection

**Comment by R-Jim** [comment]
@DuongTyler can you review this again?

**Comment by DuongTyler** [comment]
Thanks for the reminder, will check it out once I get home 👍

File: frontend/src/components/gallery/artworkCard/artworkImage.tsx [hunk 1]
```diff
@@ -58,7 +58,6 @@ export default class ArtworkImage extends React.Component<ArtworkProps, ArtworkS
     }
 
     render() {
-        console.log(this.getImage())
```
Comment (zapplecat) [inline frontend/src/components/gallery/artworkCard/artworkImage.tsx:61]: nice catch.

File: frontend/src/models/achor.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ export interface Anchor {
     href: string;
```
Comment (zapplecat) [inline frontend/src/models/achor.ts:2]: I just noticed... The name of this file is `achor` and not `anchor`

**Review by DuongTyler — APPROVED** [review]
This looks really goooood!

**Comment by DuongTyler** [comment]
ah, make sure you bring the changes over to the mass-backend repo, we are gonna retire the backend folder here

**Pull request merged**